include/jemalloc/internal/psset.h

```diff
@@ -62,6 +65,8 @@ void psset_stats_accum(psset_stats_t *dst, psset_stats_t *src);
 void psset_insert(psset_t *psset, edata_t *ps);
 void psset_remove(psset_t *psset, edata_t *ps);
 
+void psset_hugeify(psset_t *psset, edata_t *ps);
```

Comment:
`hugify` might be the better term?

Reply:
Something about that feels very wrong to me, but some googling indicates that it's by far a more popular term.